Assets/MRTK.ThirdParty/MRTK-Quest/Resources/MRTK-OculusConfig.asset

```diff
@@ -16,8 +16,9 @@ MonoBehaviour:
   ovrCameraRigPrefab: {fileID: 2343937678411072827, guid: e855e4c2a827daf45be5e0ca86dd6d05,
     type: 3}
   allowDevToManageAvatarPrefab: 0
-  localAvatarPrefab: {fileID: 1703282039130822552, guid: bc9433565454e104f9ac73600c65a612,
+  localAvatarPrefab: {fileID: 6297684789857299957, guid: bc9433565454e104f9ac73600c65a612,
```

Comment:
I notice that I often seem to have to relink the local avatar prefab in MRTK-Quest - guid is same but fileID is different - is there a meta file issue?  I'm guessing you didn't intentionally change this

Reply:
I saw you changed it with your last commit, and it broke the reference on my side. Perhaps I should reclone to see Im causing this issue.